PWGLF/TableProducer/cascadebuilder.cxx

```diff
@@ -120,7 +120,7 @@ struct cascadebuilder {
 
       std::array<float, 3> pVtx = {v0.collision().posX(), v0.collision().posY(), v0.collision().posZ()};
 
-      auto b = casc.bachelor_as<FullTracksExt>();
+      // auto b = casc.bachelor_as<FullTracksExt>();
```

Comment:
Let's just maybe remove that to have it less cluttered :-D

Reply:
I kept them in case you had written them for a later use; I will remove them if they are not needed then!